support.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Support
+page_title: Support
+description: Copyright
+slug: support-page
+tags: Fiddler support, trial, pro version, paid support
+published: True
+position: 80
+---
+
+## Fiddler Everywhere Support 
+
+ 
+#### Who has access to support? 
+
+All users who have obtained a Trial subscription or PRO plan subscription will have access to support for the duration of the active subscription period, the table below describes the comparison between different plans and level of support offered for each plan. 
```

Comment:
1) "The table below..." should be a separate sentence.
2) Consider rephrasing, to avoid the need to change later when we add more license types. For example:
"Users who have purchased a paid subscription or have activated the trial period can reach out to the Fiddler Everywhere team for help. The table below shows a comparison between the different plans and their respective level of support."